### DISCUSSION TITLE
目前我使用的是vite4.3.5+vant4，在测试进阶用法-Rem 布局适配时，使用其他设计稿尺寸未成功，寻求帮助

### DISCUSSION THREAD

**Discussion by RyanHo97** [youzan/vant#11828] [Q&A] [answered]
此处是cjs配置：
![image](https://user-images.githubusercontent.com/49436413/236785301-7d6fc37f-ba8a-4208-938c-2ba2c2eb167e.png)
此处是代码：
![image](https://user-images.githubusercontent.com/49436413/236785593-b8d3b134-919e-4ac2-a7b5-dff2c03cdb17.png)
此处是问题点，是否应该是750px正好占满屏幕？请帮助我：
![image](https://user-images.githubusercontent.com/49436413/236785656-006eebc8-6f20-4f27-883c-bdaea4005f67.png)

**Comment by RyanHo97** [comment]
我是否已经找到答案：
![image](https://user-images.githubusercontent.com/49436413/236787650-e024c644-fd3a-4e96-b716-7dcb35f86796.png)

**Reply by RyanHo97** [reply]
这个*虽然会把box宽度适配但是vant样式本身会失效，有什么更好的解决方案？

**Comment by chenjiahan** [comment, marked as answer]
没看出上面的截图有什么问题，box 确实是占满了屏幕吧

**Reply by RyanHo97** [reply]
感谢您的回复，box是占满屏幕的，但是溢出了屏幕可视区域一倍，这是否是一个问题，还是说我的用法本身不对。
![image](https://user-images.githubusercontent.com/49436413/236794066-c1ffa597-277d-417d-ae09-55c437dc7708.png)

**Reply by RyanHo97** [reply]
如果我的设计稿宽度设计为750px，是不是应该正好100%显示。
现在应该是75px=1rem，我感觉只是因为我的indexOf(这里面选中的问题)，
如果我变成了*号就可以不溢出了，但是vant组件可能会变形成这种感觉：
![image](https://user-images.githubusercontent.com/49436413/236796425-92ead263-2744-4926-baa6-58928a89c570.png)

**Reply by chenjiahan** [reply]
构建配置按 Vant 文档来配置就行，如果 box 宽度有问题的话，可以看下你的 root font size 设置的对不对。

**Reply by RyanHo97** [reply]
好的，我来检查一下

**Reply by RyanHo97** [reply]
我的root并未设置任何root font-size，我在网上按照vant3的步骤写的，他的代码同我最开始上传的图片一样但是并没溢出。我看关于vant4的文档rem布局部分也没有变化，所以也尝试用750px的vant官方文档配置方式测试，发现溢出。
这是style.css
![image](https://user-images.githubusercontent.com/49436413/236804794-f4c31c87-de45-4d8f-b0e8-3fa7fdedb521.png)

**Reply by RyanHo97** [reply]
参考链接：
https://www.bilibili.com/video/BV1TU4y177UG?p=6&vd_source=7d57c761e8c6204c0f43a06d756c834e
建议直接在第六分钟开始看：
![image](https://user-images.githubusercontent.com/49436413/236805291-5537b612-b4a9-4836-b244-7106d6aa38ba.png)

**Reply by RyanHo97** [reply]
我看到了是我的问题，他也存在这个问题：
![image](https://user-images.githubusercontent.com/49436413/236806161-72e4a40c-5311-4b11-a94c-15e5e25c4d86.png)
